README.md

```diff
@@ -9,3 +13,8 @@ might help with the notation for mathematical expressions.
 $G_1=(V_1 , E_1)$ is isomorphic to $G_2 = (V_2, E_2)$ if there exists a
 one-to-one and onto function (bijection) $f: V_1 \rightarrow V_2$ such that $(u,v)
 \in E_1$ iff $(f(u),f(v)) \in E_2$.
+By definition, a completely connected graph has an edge connecting every individual pair of nodes in the graph. Since this means every node has a direct connection to every other node, each node has n - 1 edges connected to it.
+
+Base case: With a graph containing 2 nodes that is completely connected, there is only one way for it to exist which is with 1 edge connecting the two existing nodes. Since there is only one way to create a graph of 2 nodes that is completely connected, all completely connected graphs with 2 nodes must be isomorphic.
```

Comment:
Not following you here. What about a graph with 1 node? What do you mean by "only one way to create a graph of 2 nodes that is completely connected"? And how does this relate to isomorphism, which requires a bijective mapping?